docs/android.md

```diff
@@ -57,12 +53,40 @@ To follow along this guide and build OpenDDS you will need:
  - Some knowledge about OpenDDS and Android development will be assumed, but
    more OpenDDS knowledge will be assumed than Android knowledge.
 
-In addition to those, building OpenDDS with optional dependencies also
-have requirements not listed here but will in their own sections.
+In addition to those, building OpenDDS with optional dependencies also have
+requirements not listed here but will in their own sections.
 
 The [\"Using OpenDDS in a Android App\"
 section](#using-opendds-in-a-android-app) assumes the use of Android Studio,
-but the will also work when just using the Android SDK.
+but the will also work when just using the Android SDK if tweaked.
+
+## Building on Windows
+
+If using you're using Windows Subsystem for Linux (WSL), Docker, or anything
+like that, then when you done with the guide you can copy the resulting
+libraries from the virtual environment to Windows and where they can be used in
+Android Studio as they would be used on Linux. Also you can skip the rest of
+this section and any **Windows Users:** notices.
+
+If you want to build OpenDDS for Android on Windows without any kind of
+vitalization you will have to follow the **Windows Users:** notices.
```

Comment:
```suggestion
virtualization you will have to follow the **Windows Users:** notices.
```

Reply:
> vitalization - the state of being vitalized and filled with life

No, I think that is correct (: 

I guess neovim's default spelling dictionary doesn't have "virtualization".